addons/website_slides/controllers/main.py

```diff
@@ -805,29 +805,16 @@ def slide_get_image(self, slide_id, field='image_128', width=0, height=0, crop=F
         if field not in ('image_128', 'image_256', 'image_512', 'image_1024', 'image_1920'):
             return werkzeug.exceptions.Forbidden()
 
+        if (width, height) == (0, 0):
```

Comment:
It is already done in `_get_image_stream_from` ?

Reply:
Also logic of checking only for (0, 0) seems a bit odd, I'd figure if *either* dimension is 0 it's the default or a mistake, a `1250, 0` image doesn't make any more sense.

Reply:
Resizing an image to 64x0 resizes the image to 64px in width and it uses a dynamic value for the height so that it keeps the aspect ratio. @Gorash to confirm

Testing for (0, 0) in website_slides is redundant indeed, I'm removing it.